install.md

```diff
@@ -157,18 +157,7 @@ $ git clone https://github.com/containers/skopeo $GOPATH/src/github.com/containe
 $ cd $GOPATH/src/github.com/containers/skopeo && make bin/skopeo
 ```
 
-### Building in a container
-
-Building in a container is simpler, but more restrictive:
-
-- It requires the `podman` command and the ability to run Linux containers
-- The created executable is a Linux executable, and depends on dynamic libraries
-  which may only be available only in a container of a similar Linux
-  distribution.
-
-```bash
-$ make binary # Or (make all) to also build documentation, see below.
-```
+By default the `make` comman (make all) will build bin/skopeo and the documation locally.
```

Comment:
```suggestion
By default the `make` command (make all) will build bin/skopeo and the documentation locally.
```

(Or… I don’t know; this is a bit strange place to put this. The way I think about this, either it should be the only command we document for non-container builds, or we can document the binary/docs pair and not this one. Telling users about the way to do both _after_ the first part was already done might be annoying. OTOH, bike shed colors and all that — as long as the documentation has been updated to not claim that `make all` runs in containers, I’m fine with this.